field/age/styles.css

```diff
@@ -1,5 +1,15 @@
+/*
+ * Legenda:
+ *     .path-mod-surveypro #userentry              form to preview and to submit a response;
+ *     .path-mod-surveypro #usersearch             form to search a response;
+ *     .path-mod-surveypro #userentry .frozen      readonly response
+ *     .path-mod-surveypro table#manageitems       table to manage items
+ *     .path-mod-surveypro table#sortitems         table to sort items
+ *     .path-mod-surveypro table#validaterelations table to validate relations
+ */
+
 /* Begin of: horizontal align of labels inside age elements in read-only mode */
-.age_static {
+.path-mod-surveypro #userentry .frozen .age_static {
```

Comment:
So... are you sure that you want to add `#userentry .frozen` here? That will make other elements in the Legenda to stop matching and only the 3rd one will match.

Or am I missing anything?

Reply:
To say it with other words... 

1. What is said in the Legendas (in many places in the patch) doesn't match the selectors that are applied later.
2. I've seen that you've copied the same "Legenda" (6 selectors) over and over... and it's really confusing, especially when point 1. above happens.

Reply:
Let me check it once again. Thank you for the comment.

Reply:
yes, after the creation of the usertemplate I sent you privately I can confirm that:

    having done a submission I can say that the style of the surveyprofield_age plugin appear correct;
    I was not able to find styles for subplugin like, for instance, ".path-mod-surveypro-surveyproreport-attachment". Because of this I used, for instance, #page-mod-surveypro-reports

Summarizing:

    surveypro core pages are all declared (if I am not wrong) using: .path-mod-surveypro
    surveypro subplugin pages are declared, for instance, using: #page-mod-surveypro-reports